front/src/routes/integration/all/withings/device-page/Device.jsx

```diff
@@ -96,7 +96,7 @@ class WithingsDeviceBox extends Component {
                         class="card-img-top"
                         style="aspect-ratio: 1 / 1;"
                         src={`/assets/images/withings/${WithingsDeviceImgByModel[props.device.model]}`}
-                        alt={<Text id="global.logoAlt" />}
+                        alt="/assets/icons/android-icon-192x192-round.png"
```

Comment:
The alternative text is used for non graphical browsers, I'm not sur that the hard coded image url isuseful.

https://www.w3schools.com/tags/att_img_alt.asp

Reply:
I remove this change